pdfarranger/exporter.py

```diff
@@ -75,41 +113,63 @@ def _set_meta(mdata, pdf_input, pdf_output):
         for k, v in mdata.items():
             outmeta[k] = v
 
+def _scale_box(box, factor):
+    return [factor * float(x) for x in box]
+
+
+def _fix_annots(page, factor):
+    if '/Annots' in page.keys():
+        for annot in page.Annots:
+            for key in ('/Rect', '/QuadPoints', '/Vertices', '/CL'):
+                if key in annot.keys():
+                    annot[key] = _scale_box(annot[key], factor)
+            if '/InkList' in annot.keys():
+                for i, il in enumerate(annot.InkList):
+                    annot.InkList[i] = _scale_box(il, factor)
+
 
 def _scale(doc, page, factor):
     """ Scale a page """
     if factor == 1:
         return page
-    rotate = 0
-    if "/Rotate" in page:
-        # We'll set the rotate attribute on the resulting page so we must
-        # unset it on the input page before
-        rotate = page.Rotate
-        page.Rotate = 0
+    orig_page = page
+    page = copy.copy(page)
+    page.Rotate = 0
     page = doc.make_indirect(page)
-    page_id = len(doc.pages)
+    page_id = len(doc.pages) + 1
     newmediabox = [factor * float(x) for x in page.MediaBox]
-    content = "q {} 0 0 {} 0 0 cm /p{} Do Q".format(factor, factor, page_id)
+    contents = "q {} 0 0 {} 0 0 cm /p{} Do Q".format(factor, factor, page_id)
     xobject = pikepdf.Page(page).as_form_xobject()
-    new_page = pikepdf.Dictionary(
-        Type=pikepdf.Name.Page,
-        MediaBox=newmediabox,
-        Contents=doc.make_stream(content.encode()),
-        Resources={'/XObject': {'/p{}'.format(page_id): xobject}},
-        Rotate=rotate,
-    )
-    return new_page
+    orig_page.MediaBox = newmediabox
+    #Question - if we scale a cropped page, do we loose content?
+    if '/CropBox' in orig_page.keys(): del(orig_page['/CropBox'])
```

Comment:
Scaling is relative with respect to the origin. Hence, no matter if scaling is active or not, the relative clipped content is the same.

Why is it necessary to delete the CropBox here? If it is necessary, should it moved to `_mediabox()`?